### ISSUE TITLE
[dev-env] Download translations task shouldn't create empty po files

### ISSUE THREAD

**Issue by ma2ciek** [ckeditor/ckeditor5-dev#95]
Now `translation:download` task creates ~50 files, when most of them are empty. Preventing this requires parsing po files just after download to be sure that translation file is empty.

**Comment by Reinmar** [comment]
Could you check if this is possible to do quickly? Would be great to fix it before we commit the first batch of translations.

**Comment by ma2ciek** [comment]
Yep. I should do it quickly and will add required changes to the #94 PR.
